vividus-plugin-web-app/src/test/java/org/vividus/ui/web/util/SearchAttributesConversionUtilsTests.java

```diff
@@ -67,12 +67,16 @@ void testConvertToSearchAttributesInvalidFilterType()
         assertEquals("Unsupported filter type: notFilter", exception.getMessage());
     }
 
-    @Test
-    void testConvertToSearchAttributesInvalidLocatorFormat()
+    @ParameterizedTest
+    @ValueSource(strings = {
+        "To.xpath(.a)",
+        "By.id(id):"
+    })
+    void testConvertToSearchAttributesInvalidLocatorFormat(String invalidLocator)
     {
         IllegalArgumentException exception = assertThrows(IllegalArgumentException.class,
-            () -> SearchAttributesConversionUtils.convertToSearchAttributes(INVALID_LOCATOR));
-        assertEquals(INVALID_LOCATOR_MESSAGE + INVALID_LOCATOR + CLOSING_BRACKET, exception.getMessage());
+            () -> SearchAttributesConversionUtils.convertToSearchAttributes(invalidLocator));
```

Comment:
I think since we are aware of the missing part, why not notify the user carefully?

Reply:
I think to make the message understandable to end users we should get rid of the regex in the error message and then break that regex into understandable pieces that will be clear even for people who don't know regexes; I just followed approach accepted initially and suggested improvement is better to implement in separate commit

Reply:
probably I didn't understand you correctly